b1ddi/resource_ipam_address.go

```diff
@@ -33,16 +37,18 @@ func resourceIpamsvcAddress() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Computed:     true,
+				ForceNew:     true,
 				ExactlyOneOf: []string{"address", "next_available_id"},
-				Description:  "The resource ID in the form \"/ipam/[address_block|subnet|range]/<UUID>\".",
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^ipam\/(range|subnet|address_block)\/[0-9a-f-].*$`), "invalid resource ID specified"),
+				Description:  "The resource ID in the form \"/ipam/[address_block|subnet|range]/<UUID>\". This will create the next available resource in the given container",
```

Comment:
Don't the id's start with `ipam` and not `/ipam`?

Reply:
Fixed.